app/views/goals/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Goals#create</h1>
```

Comment:
rails g probably made this file, `create.html.erb` can be safely deleted. Create actions don't usually have associated view templates.